src/python/pants/backend/jvm/tasks/jvm_compile/jvm_compile.py

```diff
@@ -133,7 +138,7 @@ def register_options(cls, register):
              help='Extra compiler args to use when warnings are disabled.')
 
     register('--fatal-warnings-enabled-args', advanced=True, type=list, fingerprint=True,
-             default=list(cls.get_fatal_warnings_enabled_args_default()),
+             default=CLASS_NOT_FOUND_ERROR_PATTERNS,
```

Comment:
Fixed. :( late night edits, remember there was sth odd. Thanks for catching this